src/elements/content-sidebar/versions/__tests__/VersionsItem.test.js

```diff
@@ -238,5 +238,111 @@ describe('elements/content-sidebar/versions/VersionsItem', () => {
             expect(wrapper.exists(VersionsItemRetention)).toBe(true);
             expect(wrapper.find(VersionsItemActions).prop('isRetained')).toBe(true);
         });
+
```

Comment:
Can you please rewrite those test to use React Testing Library. Enzyme is a huge problem in EUA and we need to slowly convert existing tests instead of creating new ones.

Reply:
Updated to RTL